keycloak/keycloak_client.go

```diff
@@ -122,6 +122,9 @@ func (keycloakClient *KeycloakClient) login() error {
 	if err != nil {
 		return err
 	}
+	if accessTokenResponse.StatusCode != 200 {
```

Comment:
nitpick: let's use `http.StatusOk` here

```suggestion
	if accessTokenResponse.StatusCode != http.StatusOk {
```

Reply:
Good point, I updated the PR.